backends/tfhe-cuda-backend/cuda/include/helper_multi_gpu.h

```diff
@@ -8,7 +8,7 @@ extern std::mutex m;
 extern bool p2p_enabled;
 
 extern "C" {
-int cuda_setup_multi_gpu();
+int32_t cuda_setup_multi_gpu();
```

Comment:
For curiosity, why do we want to have `int32_t` instead of `int` here?

Reply:
Just to be sure with the types on any architecture, I prefer to always specify the size of the integer (as we do on the Rust side as well).